charts/delphai-deployment/templates/deployment.yaml

```
@@ -210,6 +210,13 @@ spec:
             {{ end }} {{/* if ne $portName "oauthproxy" */}}
             {{ end }} {{/* range ports */}}
 
+            {{ if $.Values.monitoring.enabled }}
+            {{ range $endpoints := .Values.monitoring.podMetricsEndpoints }}
+            - name: "{{ $endpoints.targetPort }}-metrics"
```

Does it work fine with a digits-prefixed name? Dangerous identifier
Lets maybe have 
```suggestion
            - name: "metrics-{{ $endpoints.targetPort }}"
```
?

---

Yeah It was fine, but I agree this is better, thanks 👍